<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Binance Futures Trading</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>8</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="topWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>600</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSplitter" name="topSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>8</number>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="leftWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>1080</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="chartContainer"/>
          </widget>
          <widget class="QWidget" name="rightWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>450</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="userLayout">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="userButton">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>User</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="logoutButton">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Đăng xuất</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="symbolLayout">
              <item>
               <widget class="QLabel" name="symbolLabel">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Cặp giao dịch:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="symbolComboBox">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="priceBalanceLayout">
              <item>
               <layout class="QVBoxLayout" name="balanceLayout">
                <item>
                 <widget class="QLabel" name="balanceLabel">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font-weight: bold; font-size: 16px;</string>
                  </property>
                  <property name="text">
                   <string>Số dư:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="balanceValueLabel">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(0, 0, 0, 0.05); border-radius: 5px; padding: 8px; font-size: 16px;</string>
                  </property>
                  <property name="text">
                   <string>0.0</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::RichText</enum>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QFrame" name="tradingSettingsFrame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="autoTradingLabel">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Giao dịch tự động:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="autoTradingCheckBox">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="timeframeLabel">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Khung thời gian:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="timeframeComboBox">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="amountLabel">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Số tiền giao dịch (USDT):</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="amountSpinBox">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="minimum">
                  <double>10.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>10000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>100.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="leverageLabel">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Đòn bẩy:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSpinBox" name="leverageSpinBox">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>125</number>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="stopLossLabel">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Stop loss :</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="takeProfitLabel">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Take Profit:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QDoubleSpinBox" name="takeProfitSpinBox">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="minimum">
                  <double>0</double>
                 </property>
                 <property name="maximum">
                  <double>999999999.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="specialValueText">
                  <string></string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QDoubleSpinBox" name="stopLossSpinBox">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="minimum">
                  <double>0</double>
                 </property>
                 <property name="maximum">
                  <double>999999999.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="specialValueText">
                  <string></string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0" colspan="2">
                <layout class="QHBoxLayout" name="buySellLayout">
                 <item>
                  <widget class="QPushButton" name="buyButton">
                   <property name="font">
                    <font>
                     <pointsize>13</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: #00C853; color: white; font-weight: bold; font-size: 14px; padding: 8px;</string>
                   </property>
                   <property name="text">
                    <string>MUA</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="sellButton">
                   <property name="font">
                    <font>
                     <pointsize>13</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: #FF3D00; color: white; font-weight: bold; font-size: 14px; padding: 8px;</string>
                   </property>
                   <property name="text">
                    <string>BÁN</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bottomWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>350</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="tradeLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Vị thế đang mở</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tradeTable">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>210</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="bottomControlsWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="summaryLayout">
             <item>
              <widget class="QLabel" name="totalProfitLabel">
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Tổng lợi nhuận: 0.00 USDT</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="winRateLabel">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Tỷ lệ thắng: 0.00%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lastUpdateLabel">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Cập nhật lần cuối:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="filterLayout">
             <item>
              <widget class="QLabel" name="filterLabel">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Lọc:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="filterComboBox">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>Tất cả</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Đang mở</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Đã đóng</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ứng dụng</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Binance</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>